web/src/icons/index.tsx

```diff
@@ -0,0 +1,142 @@
+import clsx from "clsx";
+import React from "react";
+import { tw } from "../lib/tailwind";
+import AspectCabinetry from "./aspect-cabinetry.svg";
+import Best from "./best.svg";
+import Bird from "./bird.svg";
+import Check from "./check.svg";
+import Ckd from "./ckd.svg";
+import Construction from "./construction.svg";
+import Design from "./design.svg";
+import EclipseCabinetry from "./eclipse-cabinetry.svg";
+import Facebook from "./facebook.svg";
+import Heart from "./heart.svg";
+import Houzz from "./houzz.svg";
+import Instagram from "./instagram.svg";
+import JKCabinetry from "./j-k-cabinetry.svg";
+import Layout from "./layout.svg";
+import LeftArrow from "./left-arrow.svg";
+import Logo from "./logo.svg";
+import MagnifyingGlass from "./magnifying-glass.svg";
+import Material from "./material.svg";
+import Nkba from "./nkba.svg";
+import Remodel from "./remodel.svg";
+import Rendering from "./rendering.svg";
+import Shield from "./shield.svg";
+import ShilowCabinetry from "./shilow-cabinetry.svg";
+
+type Icon = React.FC<{
+  className?: string;
+}>;
+
+const baseClass = tw`p-1 fill-current stroke-current inline-block`;
+const iconBaseClass = tw`fill-none stroke-3`;
+const roundedBaseClass = tw`rounded-full`;
+
+// Bluebird
+export const BirdIcon: Icon = ({ className }) => (
+  <Bird className={clsx(baseClass, className)} />
```

Comment:
`clsx` lets us add class names together and throw out falsy values like `undefined`